content/en/docs/instrumentation/java/automatic/servers.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Configuring application servers
+description: Learn how to define agent paths for Java application servers
+weight: 15
+cSpell:ignore: asadmin binsetenv binstart Glassfish Payara setenv
+---
+
+When instrumenting an application running on an application server, you've to
+define the path to the OpenTelemetry Java agent. The way to define the path
+differs from server to server.
+
+## JBoss EAP / WildFly
+
+To define the path to the Java agent, add the `javaagent` argument at the end of
+the `standalone.conf` or the `standalone.conf.bat` configuration files:
+
+```shell
+# Linux / standalone.conf
+JAVA_OPTS="$JAVA_OPTS -javaagent:/path/to/opentelemetry-javaagent.jar"
+
+# Windows / standalone.conf.bat
+set "JAVA_OPTS=%JAVA_OPTS% -javaagent:<Drive>:\path\to\opentelemetry-javaagent.jar"
+```
```

Comment:
You'll want to use tabs for a better UX (in particular because tab choice is persisted):

~~~suggestion
{{< tabpane text=true >}} {{% tab header="Linux (standalone.conf)" lang=Linux %}}

```sh
JAVA_OPTS="$JAVA_OPTS -javaagent:/path/to/opentelemetry-javaagent.jar"
```

{{% /tab %}} {{% tab header="Windows (standalone.conf.bat)" lang=Windows %}}

```bat
set "JAVA_OPTS=%JAVA_OPTS% -javaagent:<Drive>:\path\to\opentelemetry-javaagent.jar"
```

{{% /tab %}} {{< /tabpane >}}
~~~

This will render as:

> <img width="686" alt="image" src="https://github.com/open-telemetry/opentelemetry.io/assets/4140793/5ca4f834-d067-4e46-8161-5f18f03ead1c">

**NOTE**: you won't need the `tab` attributes `header` and `lang` for _other_ tabbed panes. See below for a simplified example.